### PR TITLE
Update package.json

### PR DESCRIPTION
**Pull request by etaletai13**
handle bower deps in a build step for rebuilding bower deps upon yarn add/install as dep itself.

-- this removes the need to rely on bower at all in the main app

cleanly works with  https://github.com/Musicoin/desktop/pull/252

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -4,9 +4,10 @@
   "license": "MIT",
   "private": true,
   "scripts": {
-    "install": "yarn bower-installer -p",
-    "test": "echo 'No tests yet.'"
-  },
+    "build": "bower-installer -p",
+    "install": "bower-installer -p",
+    "test": "yarn build"
+  }
```
Comment (Varunram) [inline package.json]: @etaletai13 you missed the usual comma here :)

**Review by Varunram — CHANGES_REQUESTED** [review]

**Review by Varunram — COMMENTED** [review]
Assuming this works in tandem with there there merged PR.

**Comment by Varunram** [comment]
weirdly enough, I still find it convenient to use `bower install`, maybe its a matter of practice before I shift to `yarn` entirely..

**Comment by etaletai13** [comment]
@Varunram I really understand that, actually! I love that about yarn, because I can actually completely shift over in theory but still wrap a script around so that `bower install` ends up performing the right actions anyway. Yarn applied to things it wasn't built for is what I like to think of as crochet; because why  not push the metaphor into parable land? Anyway, this is forward moving. As I've learned, it behoves me to keep the branch around for quite a while really, just in case. I tend to be the reverse of a hoarder, and am learning to not erase the past every two minutes!! Let's keep going.

**Pull request merged**